index.html

```diff
@@ -1,39 +1,230 @@
+            <h4>Ruby on rails</h4>
+            <h4>CSS</h4>
+            <h4>JavaScript</h4>
+            <h4>HTML</h4>
+          </div>
+          <button type="button">See Project</button>
+        </div>
+      </div>
+
+      <div class="grid-item">
+        <img src="https://via.placeholder.com/300x250" alt="imagePlaceholder" />
+        <div class="description-container">
+          <h3>Multi-Post Stories Gain+Glory</h3>
+          <div class="tags">
+            <h4>Ruby on rails</h4>
+            <h4>CSS</h4>
+            <h4>JavaScript</h4>
+            <h4>HTML</h4>
+          </div>
+          <button type="button">See Project</button>
+        </div>
+      </div>
+
+      <div class="grid-item">
+        <img src="https://via.placeholder.com/300x250" alt="imagePlaceholder" />
+        <div class="description-container">
+          <h3>Multi-Post Stories Gain+Glory</h3>
+          <div class="tags">
+            <h4>Ruby on rails</h4>
+            <h4>CSS</h4>
+            <h4>JavaScript</h4>
+            <h4>HTML</h4>
+          </div>
+          <button type="button">See Project</button>
+        </div>
+      </div>
+
+      <div class="grid-item">
+        <img src="https://via.placeholder.com/300x250" alt="imagePlaceholder" />
+        <div class="description-container">
+          <h3>Multi-Post Stories Gain+Glory</h3>
+          <div class="tags">
+            <h4>Ruby on rails</h4>
+            <h4>CSS</h4>
+            <h4>JavaScript</h4>
+            <h4>HTML</h4>
+          </div>
+          <button type="button">See Project</button>
+        </div>
+      </div>
+
+      <div class="grid-item">
+        <img src="https://via.placeholder.com/300x250" alt="imagePlaceholder" />
+        <div class="description-container">
+          <h3>Multi-Post Stories Gain+Glory</h3>
+          <div class="tags">
+            <h4>Ruby on rails</h4>
+            <h4>CSS</h4>
+            <h4>JavaScript</h4>
+            <h4>HTML</h4>
+          </div>
+          <button type="button">See Project</button>
+        </div>
+      </div>
+
+      <div class="grid-item">
+        <img src="https://via.placeholder.com/300x250" alt="imagePlaceholder" />
+        <div class="description-container">
+          <h3>Multi-Post Stories Gain+Glory</h3>
+          <div class="tags">
+            <h4>Ruby on rails</h4>
+            <h4>CSS</h4>
+            <h4>JavaScript</h4>
+            <h4>HTML</h4>
+          </div>
+          <button type="button">See Project</button>
+        </div>
+      </div>
     </div>
-  </body>
-</html>
+  </div>
+
+  <!-- About Me Section -->
+  <section class="aboutMeSection">
+    <h2 class="aboutMe">About Me</h2>
+    <p class="aboutMeText"> I can help you build a product, feature or website Look through some of my work and
+      experience! If you like what you see and have a project you need coded, don't hesitate to contact me.
+    </p>
+    <button type="button">Get My Resume</button>
+  </section>
```

Comment:
- [ ] Please fix the background images for the about section, They are overflowing on content.

### Your design

![image](https://user-images.githubusercontent.com/83803414/158867290-c4719980-df00-4817-a90e-4c68bd0e4ce3.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/158867330-b4ff3512-c8fd-4ffe-bb60-90a436f68423.png)